apps/els_lsp/test/els_code_action_SUITE.erl

```diff
@@ -53,14 +54,18 @@ init_per_testcase(TestCase, Config) ->
 -spec end_per_testcase(atom(), config()) -> ok.
 end_per_testcase(TestCase, Config) ->
   els_test_utils:end_per_testcase(TestCase, Config).
+%%==============================================================================
+%% Const
+%%==============================================================================
+-define(COMMENTS_LINES, 2).
```

Comment:
Clever. This offset-based approach is something we could use in many other test suites...